meson.build

```diff
@@ -1199,6 +1199,20 @@ endif
 
 ############################################################
 
+opt = not get_option('build-bits').require(is_glibc).disabled()
```

Comment:
Why does it require glibc?
A feature test would be better in any case.

Reply:
That is an excellent question. I only copied what I found for another command.

What would be the best way to determine if it should be built or not? I don't rely on many libc features, I think.

Reply:
Just delete the dependency and let the CI figure it out.

Reply:
Done :+1: